linkerd/docs/namer.md

```diff
@@ -187,6 +187,7 @@ token | no authentication | The auth token to use when making API calls.
 setHost | `false` | If `true`, HTTP requests resolved by Consul will have their Host header overwritten to `${serviceName}.service.${datacenter}.${domain}`. `$domain` is fetched from Consul.
 consistencyMode | `default` | Select between [Consul API consistency modes](https://www.consul.io/docs/agent/http.html) such as `default`, `stale` and `consistent`. 
 failFast | `false` | If `false`, disable fail fast and failure accrual for Consul client. Keep it `false` when using a local agent but change it to `true` when talking directly to an HA Consul API 
+localDcMoniker | none | If set, can be used to reference a datacenter of the agent the namer is connected to. E.g., if connected to an agent in datacenter `dc1` given `localDcMoniker: _local` name `/#/io.l5d.consul/_local/myservice` is equal to `/#/io.l5d.consul/dc1/myservice` 
```

Comment:
Well, I don't mind hardcoding `.local` (as I don't see how that could be a valid DC name in Consul) - I though you won't like that. :) So, should I remove this option then and update the doc?